alicorn-expressions.lua

```diff
@@ -533,7 +534,7 @@ local function operative_test_hack(env, metaval)
 	end
 end
 
----@param start_anchor Anchor
+---@param span Span
```

Comment:
Please describe what `span` represents / is associated with, in either the argument name or the argument documentation.